MechJeb2/MechJebModuleStagingController.cs

```diff
@@ -79,9 +79,9 @@ public override void OnUpdate()
             if (!vessel.isActiveVessel)
                 return;
 
-            //if autostage enabled, and if we are not waiting on the pad, and if there are stages left,
+            //if autostage enabled, and if we've already staged at least once, and if there are stages left,
             //and if we are allowed to continue staging, and if we didn't just fire the previous stage
-            if (!vessel.LiftedOff() || StageManager.CurrentStage <= 0 || StageManager.CurrentStage <= autostageLimit
+            if (StageManager.CurrentStage == StageManager.StageCount || StageManager.CurrentStage <= 0 || StageManager.CurrentStage <= autostageLimit

```

Comment:
so this relies on the fact that MJ has "KSP stages plus one" stages?  that'll probably work, and likely fixes "the MET bug" but seems a little side-effect-ey...